escrow/token_service.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package escrow;
+
+option java_package = "io.singularitynet.daemon.escrow";
+//It is expected that the user would call the GetChannelState to Determine the Current state of the Channel
+//Based on the usage forecast, the user/client will have to sign for an amount L + U , where L is the last amount Signed
+//and U is the amount based on expected usage.
+//Please be aware that the Signing up an amount upfront ( Pre Paid)  does come with a risk and hence the
+//user must exercise caution on the amount signed specially with new service providers.
+//If there is no need of making concurrent calls then you may consider pay per mode.
+//Using a Token, the Client can now make concurrent calls, which was not supported previously with the pay per mode.
+//However the pay per mode is a lot secure than the pre-paid mode.
+service TokenService {
+    // GetToken method checks the Signature sent and returns a Token
+    // 1) The Signature is valid and has to be signed in the below format
+    //"__MPE_claim_message"+MpeContractAddress+ChannelID+ChannelNonce+SignedAmount
+    //Signature is to let the Service Provider make a claim
+    // 2) Signed amount >= Last amount Signed.
+    //planned amount 10 , usage 5 , this signed 10 cogs
+    //  if Signed amount == Last Signed amount , then check if planned_amount < used_amount
+    //  if Signed amount > Last Signed amount , then update the planned amount = planned amount + Signed Amount
+    //  if Signed amount < Last Signed amount , then an error is returned back stating invalid amount
+    // GetToken method in a way behaves as a renew Token too!.
+    rpc GetToken(TokenRequest) returns (TokenReply) {}
+
+
+
+}
+
+// TokenRequest is a request for getting a valid token.
+message TokenRequest {
+    // channel_id contains id of the channel which state is requested.
+    uint64 channel_id = 1;
+    // current_nonce is a latest nonce of the payment channel.
+    uint64 current_nonce = 2;
+    //signed_amount is the amount signed by client with current_nonce
+    uint64 signed_amount = 3;
+    // signature is a client signature of the message which contains
+    //"__MPE_claim_message"+MpeContractAddress+ChannelID+ChannelNonce+SignedAmount
+    bytes signature = 4;
+
+}
+
+// TokenReply message contains a latest channel state. current_nonce and
+message TokenReply {
+    // current_nonce is a latest nonce of the payment channel.
+    uint64 channel_id = 1;
+
+    // current_signed_amount is a last amount which were signed by client with current_nonce
```

Comment:
Looks like copy-pasted comment here and line below.

Reply:
fixed